rna_cd/cli.py

```diff
@@ -0,0 +1,186 @@
+                   "Default = chrM")
+@click.option("-pd", "--positives-dir",
+              type=click.Path(exists=True, dir_okay=True,
+                              file_okay=False, readable=True),
+              callback=directory_callback,
+              help="Path to directory containing positive BAM files. "
+                   "Mutually exclusive with --positives-list")
+@click.option("-nd", "--negatives-dir",
+              type=click.Path(exists=True, dir_okay=True,
+                              file_okay=False, readable=True),
+              callback=directory_callback,
+              help="Path to directory containing negative BAM files. "
+                   "Mutually exlusive with --negatives-list")
+@click.option("-pl", "--positives-list",
+              type=click.Path(exists=True, dir_okay=False,
+                              file_okay=True, readable=True),
+              callback=list_callback,
+              help="Path to file containing a list of paths to positive BAM "
+                   "files. Mutually exclusive with --positives-dir")
+@click.option("-nl", "--negatives-list",
+              type=click.Path(exists=True, dir_okay=False,
+                              file_okay=True, readable=True),
+              callback=list_callback,
+              help="Path to file containing a list of paths to negative BAM "
+                   "files. Mutuallly exclusive with --negatives-dir")
+@click.option("--cross-validations", type=click.INT, default=3,
+              help="Number of folds for cross validation run. Default = 3")
+@click.option("--verbosity", type=click.INT, default=1,
+              help="Verbosity value for cross validation step. Default = 1")
+@click.option("-j", "--cores", type=click.INT, default=1,
+              help="Number of cores to use for processing of BAM files "
+                   "and cross validations. Default = 1")
+@click.option("--plot-out", type=click.Path(writable=True),
+              help="Optional path to PCA plot.")
+@click.option("-o", "--model-out", type=click.Path(writable=True),
+              required=True,
+              help="Path where model will be stored.")
+def train_cli(chunksize: int, contig: str, model_out: Path,
+              positives_dir: Optional[List[Path]] = None,
+              negatives_dir: Optional[List[Path]] = None,
+              positives_list: Optional[List[Path]] = None,
+              negatives_list: Optional[List[Path]] = None,
+              cross_validations: int = 3,
+              verbosity: int = 1, cores: int = 1,
+              plot_out: Optional[str] = None):
+
+    if positives_dir is None and positives_list is None:
+        raise ValueError("Must set either --positives-dir or --positives-list")
+
+    if negatives_dir is None and negatives_list is None:
+        raise ValueError("Must set either --negatives-dir or --negatives-list")
+
+    positives = positives_dir if positives_dir is not None else positives_list
```

Comment:
I'm a big opponent of using `None`s as boolean values. This can easily lead to bugs if you are actually expecting a boolean (not the case here, but still). Explicit is better than implicit.